### PR TITLE
Feature#22 Deploying to Heroku

### PR DESCRIPTION
**Pull request by luisgarciaes**
# Description
Fixed multiple errors for correct deployment on Heroku

### CODE REVIEW

**Review by OwenLobato — DISMISSED** [review]
lgtm

**Review by AgustinPalmaM — APPROVED** [review]
LGTM

File: Gemfile [hunk 1]
```diff
@@ -95,4 +92,8 @@ gem "letter_opener", group: :development
 gem 'jquery-rails', '~> 4.5'
 gem 'rails-ujs', '~> 0.1.0'
 
-gem 'pg_search', '~> 2.3'
\ No newline at end of file
+gem 'pg_search', '~> 2.3'
+
+source "https://rails-assets.org"
+
+gem 'bootstrap', '~> 5.1.3'
```
[the remark this answers is not in the document]
Comment (ruruisu) [reply Gemfile]: done!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by OwenLobato — APPROVED** [review]

**Review by lucasvidela94 — APPROVED** [review]
LGTM! 🎸

**Pull request merged**